overlord/ifacestate/ifacemgr_test.go

```diff
@@ -113,7 +139,14 @@ func (s *interfaceManagerSuite) TestConnectTask(c *C) {
 	ts, err := ifacestate.Connect(s.state, "consumer", "plug", "producer", "slot")
 	c.Assert(err, IsNil)
 
-	task := ts.Tasks()[0]
+	i := 0
+	task := ts.Tasks()[i]
+	c.Check(task.Kind(), Equals, "run-hook")
```

Comment:
Can you please check that each of those tasks carries the right meta-data about what hook to run?